src/Emulator/Cores/Arm-M/NVIC.cs

```diff
@@ -174,6 +174,9 @@ public void WriteDoubleWord(long offset, uint value)
                 {
                     SetPendingIRQ(14);
                 }
+                if ((value & (1u << 31)) !=0) {
+                    SetPendingIRQ(2);
```

Comment:
Verified that NMI is IRQ # 2: https://developer.arm.com/documentation/dui0552/a/the-cortex-m3-processor/exception-model/vector-table